qa/sc_node_termination_during_sync.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python3
+
+import logging
+import time
+
+from SidechainTestFramework.sc_boostrap_info import SCNodeConfiguration, SCCreationInfo, MCConnectionInfo, \
+    SCNetworkConfiguration, SC_CREATION_VERSION_2, DEFAULT_API_KEY
+from SidechainTestFramework.sc_test_framework import SidechainTestFramework
+from SidechainTestFramework.scutil import assert_true, bootstrap_sidechain_nodes, start_sc_nodes, generate_next_blocks, \
+    connect_sc_nodes, start_sc_node, wait_for_sc_node_initialization
+from httpCalls.block.best import http_block_best
+from httpCalls.transaction.createCoreTransaction import http_create_core_transaction
+from httpCalls.transaction.sendCoinsToAddress import sendCointsToMultipleAddress
+from httpCalls.transaction.sendTransaction import sendTransaction
+from httpCalls.wallet.allBoxesOfType import http_wallet_allBoxesOfType
+from httpCalls.wallet.createPrivateKey25519 import http_wallet_createPrivateKey25519
+from test_framework.util import start_nodes, \
+    websocket_port_by_mc_node_index, forward_transfer_to_sidechain, assert_equal, fail
+
+"""
+Verify successful node termination during active sync
+Configuration:
+    Start 1 MC node and 2 SC node).
+Test:
+    Create 997 blocks (this number is not important)
+    Connect sc node 2 to sc node 1
+    When sync process started, stop sc node 2
+    Verify node is terminated
+    Restart sc node 2, reconnect it to sc node 2
+    Verify that node is able to sync all the remaining blocks
+"""
+
+
+class BigBlockTest(SidechainTestFramework):
```

Comment:
let's rename the class.
Moreover add this test to the utxo list in `run_sc_tests.sh`